modules/oadp-usecase-backup-odf.adoc

```diff
@@ -0,0 +1,190 @@
++
+.Example output
+[source,terminal]
+----
+# BUCKET_NAME
+backup-c20...41fd
+# BUCKET_PORT
+443
+# BUCKET_REGION
+
+# BUCKET_SUBREGION
+
+# BUCKET_HOST
+s3.openshift-storage.svc
+----
+
+. To get the bucket credentials from the generated `secret`, run the following command:
++
+[source,terminal]
+----
+$ oc extract --to=- secret/test-obc # <1>
+----
+<1> `test-obc` is the name of the OBC.
++
+.Example output
+[source,terminal]
+----
+# AWS_ACCESS_KEY_ID
+ebYR....xLNMc
+# AWS_SECRET_ACCESS_KEY
+YXf...+NaCkdyC3QPym
+----
+
+. Get the public URL for the S3 endpoint from the s3 route in the `openshift-storage` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc get route s3 -n openshift-storage
+----
+
+. Create a `cloud-credentials` file with the object bucket credentials as shown in the following command. 
++
+[source,terminal]
+----
+[default]
+aws_access_key_id=<AWS_ACCESS_KEY_ID> # <1>
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY> # <2>
+----
+<1> Specify the AWS_ACCESS_KEY_ID.
+<2> Specify the AWS_SECRET_ACCESS_KEY.
+
+. Create the `cloud-credentials` secret with the `cloud-credentials` file content as shown in the following command.
++
+[source,terminal]
+----
+$ oc create secret generic \
+  cloud-credentials \
+  -n openshift-adp \
+  --from-file cloud=cloud-credentials
+----
+
+. Configure the DPA as shown in the following example:
++
+.Example DPA
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: oadp-backup
+  namespace: openshift-adp
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift
+        - csi
+      defaultSnapshotMoveData: true
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: noobaa
+          s3Url: https://s3.openshift-storage.svc # <1>
+          s3ForcePathStyle: "true" 
+          insecureSkipTLSVerify: "true"
+        provider: aws
+        default: true
+        credential:
+          key: cloud
+          name:  cloud-credentials 
+        objectStorage:
+          bucket: <bucket_name> # <2>
+          prefix: oadp
+----
+<1> The S3 URL of {odf-short} storage.
+<2> Specify the bucket name.
+
+. Configure a backup CR as shown in the following example:
++
+.Example backup CR
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: test-backup
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - <application_namespace> # <1>
+----
+<1> Specify the namespace for the application to back up.
+
+. Create the backup CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <backup_cr_filename> # <1>
+----
+<1> Specify the backup CR file name.
+
+.Verification
+
+. Verify that the backup object is in the `Completed` phase by running the following command.
++
+[source,terminal]
+----
+$ oc describe backup test-backup # <1>
```

Comment:
can we give an example for new-users where to look exactly in the CR ?

Reply:
Added an example output.